orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/providers/aws/AwsDeployStagePreProcessor.groovy

```diff
@@ -50,7 +51,7 @@ class AwsDeployStagePreProcessor implements DeployStagePreProcessor {
   @Override
   List<StepDefinition> additionalSteps(Stage stage) {
     def stageData = stage.mapTo(StageData)
-    if (stageData.strategy == "rollingredblack") {
+    if (Strategy.fromStrategy(stageData.strategy) == Strategy.ROLLING_RED_BLACK) {
```

Comment:
`Strategy.fromStrategy` sounds tautological to me, I would prefer `Strategy.fromString`

Reply:
yeah, this was here already... but i agree. How about `fromStrategyKey`

Reply:
I like that `Strategy.fromString` does what it says. You have a `String` and you want a `Strategy` 🤷‍♂